src/components/Verse/OverflowVerseActionsMenuBody.tsx

```diff
@@ -231,21 +95,15 @@ const OverflowVerseActionsMenuBody: React.FC<Props> = ({
         onActionTriggered={onActionTriggered}
       />
 
-      <PopoverMenu.Item
-        onClick={onToggleBookmarkClicked}
-        icon={bookmarkIcon}
-        isDisabled={isVerseBookmarkedLoading}
-      >
-        {isVerseBookmarked ? `${t('bookmarked')}!` : `${t('bookmark')}`}
-      </PopoverMenu.Item>
+      <BookmarkAction
+        verse={verse}
+        isTranslationView={isTranslationView}
+        onActionTriggered={onActionTriggered}
+        bookmarksRangeUrl={bookmarksRangeUrl}
+      />
+      <SaveToCollectionAction verse={verse} bookmarksRangeUrl={bookmarksRangeUrl} />
```

Comment:
This needs to be conditionally rendered only when the user is logged in. Another idea is to encourage people to signin/up so when they click on it, we show a toast saying please sign in to use this feature.